Components/Pages/Goals.js

```diff
@@ -64,10 +69,12 @@ const Goals = ({ navigation }) => {
       }
       if (finalStatus !== "granted") {
         // alert("Failed to get push token for push notification!");
+
         return;
       }
       let token = await Notifications.getExpoPushTokenAsync();
```

Comment:
it's interesting that this isn't under if finalStatus === "granted"  since we'd only ask for the token if access is granted. It seems like we're gong to get the token regardless if they agreed. With that being said, I read the [documentation](https://snack.expo.io/@cruzc09/c10b13?platform=android&name=Push%20Notifications&sdkVersion=37.0.0&dependencies=expo-constants%2Cexpo-permissions&sourceUrl=https%3A%2F%2Fdocs.expo.io%2Fstatic%2Fexamples%2Fv37.0.0%2Fpushnotifications.js) and that's how they have it